environment.yml

```diff
@@ -6,8 +6,7 @@
 name: RAiDER
 channels:
  - conda-forge
- - anaconda
- - defaults
+ - nodefaults
```

Comment:
Generally, it's best not to mix conda-forge and the defaults, so this enforces conda-forge only (which works!).